.github/workflows/publish-branch.yml

```diff
@@ -23,3 +23,9 @@ jobs:
     - run: npm publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+    - run: npm run public-docs
+    - name: Deploy Docs
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./out
```

Comment:
This is not actually the workflow you want. This is sort of a fall-back quick publish as needed workflow. Which maybe we don't really need as we can always manually publish. Anyways, this is not ran as the part of a standard release.

The create-release workflow is the one you want. It creates a tag, publishes to npm and then creates a GitHub release. I'd prob add this after the release creation in the workflow.

Reply:
Ah, great. Thanks for the clarification. I'll move the job around.